csharp/README.md

```diff
@@ -1,206 +1,45 @@
-# Building Ice for .NET
+# Ice for .NET
 
-This page describes how to build Ice for .NET from source and package the
-resulting binaries. As an alternative, you can download and install the
-[zeroc.ice.net][1] NuGet package.
+[Getting started] | [Examples] | [NuGet packages] | [Documentation] | [Building from source]
```

Comment:
I believe there is only one NuGet package, zeroc.ice.net.

Reply:
This like is to our ZeroC NuGet account listing all of our packages.

Reply:
Fixed